mqttClient.js

```
@@ -0,0 +1,81 @@
+const os = require('os');
+
+const logger = winston.createLogger({
+  format: winston.format.json(),
+  transports: [new winston.transports.Console()]
+});
+
+const machineInfo = os.hostname() + os.type() + os.release();
+const uniqueID = crypto.createHash('sha256').update(machineInfo).digest('hex');
+
+const MQTT_BROKER = process.env.MQTT_BROKER;
+const MQTT_PORT = parseInt(process.env.MQTT_PORT);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)